hive/utils/rliable.py

```diff
@@ -0,0 +1,323 @@
+      ax[j, i].grid(axis='y', alpha=0.1)
+  return fig
+
+def get_rliable_parameters(num_eval_episodes,
+                           num_runs_aggregates,
+                           env_names,agent_names,
+                           path,
+                           test_agent,
+                           Normalization=True,
+                           min_env_score=None,
+                           max_env_score=None,
+                           plot=False):
+  """
+  Args:
+            num_eval_episodes: No of Evaluation Episodes
+            env_names: List of names of all the environments
+            agent_names: List of names of all the agents/algorithms
+            path: Path to the folder after where the rewards/scores are stored.
+            
+            The directory structure after above given path
+            must be as follows:
+          ├── Agent 1
+          |  ├──Environment 1/
+          |   ├──Rewards
+          |  ├──Environment 2/
+          |   ├──Rewards
+          |  .
+          |  .
+          |  .
+          |  .
+          ├── Agent 2
+          |  ├──Environment 1/
+          |   ├──Rewards
+          |  ├──Environment 2/
+          |   ├──Rewards
+          |  .
+          |  .
+          |  .
+          |  .   
+          ├── Agent 3
+          |  ├──Environment 1/
+          |   ├──Rewards
+          |  ├──Environment 2/
+          |   ├──Rewards
+          |  .
+          |  .
+          |  .
+          |  .            
+
+          Here Agent 1,Agent 2,Agent 3 are the names of the algorithms provided in the agent_names parameters
+          and Environment 1,Environment 2,Environment 3 are the names of the environments provided in the env_names parameters
+          Rewards for all runs must be in form of a single numpy array.
+
+          Normalization (bool): Indicates whether the normalisation is applied or not.
+          min_env_score(float): Minimum value if normalisation is applied.
+          max_env_score(float): Maximum value if normalisation is applied.
+          num_runs_aggregates(int): Number of runs to be considered for aggregation. 
+          plot(bool): Indicates whether the reliable parameters are to be plotted or not.
+  """
+  
+  
+  
+  #Aggregates on environment (with default 10 runs)
+  if num_runs_aggregates>num_eval_episodes:
+    raise 'num_runs_aggregates should be less than or equal to num_eval_episodes'
+
+  
+  score_data_dict = {}
+  for agent_name in agent_names:
+    path_temp = path+'/'+agent_name
+    score_data_dict[agent_name] = load_and_read_rewards(env_names,path_temp,min_env_score,max_env_score,Normalization)
+
+  #Aggregates
+  aggregates_score_dict = {key: val[:num_runs_aggregates] for key, val in score_data_dict.items()}
+  aggregate_func = lambda x: np.array([MEDIAN(x), IQM(x), MEAN(x), OG(x)])
+  aggregate_scores, aggregate_interval_estimates = rly.get_interval_estimates(aggregates_score_dict, aggregate_func, reps=50000)
+
+  if plot:
```

Comment:
Hold off on this for now. We can add it once we have a good way of getting the raw metrics